Algorithms/is_unique.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.1
+# Implement an algorithm to determine if a string has all unique characters.
+
+
+def is_unique(string):
+    chars = set(string)
```

Comment:
There is no value of turning this into a set first. You're immediately returning when you find a duplicate, so you might as well go through every character.